app/src/main/java/org/sopt/at/core/util/extension/ModifierExt.kt

```diff
@@ -0,0 +1,28 @@
+package org.sopt.at.core.util.extension
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.input.pointer.pointerInput
+
+@Composable
+inline fun Modifier.noRippleClickable(crossinline onClick: () -> Unit): Modifier = composed {
+    this.clickable(
+        indication = null,
+        interactionSource = remember { MutableInteractionSource() },
+        onClick = { onClick() }
```

Comment:
불필요하게 람다를 한 번 더 감싸고 있는 거 같은데, 람다 인스턴스 생성을 줄여 미세하게 퍼포먼스를 개선할 수 있고 () -> Unit으로 고정되어 있기 때문에, 직접 참조 방식으로 수정은 어떨까요?
```suggestion
        onClick = onClick
```

Reply:
히히 놓쳤네요 수정할게요~

Reply:
아하 저 익스텐션 함수가 inline 펑션이라서 직접참조로 바꾸게 되면 람다가 인라인되지 않고 일반 함수 객체로 전달됩니다. noinline으로 바꾸면 인라인의 성능상 이점이 없어져서요~